protocol-designer/src/components/StepEditForm/fields/WellSelectionField/WellSelectionModal.tsx

```diff
@@ -121,9 +125,8 @@ const WellSelectionModalComponent = (
 export const WellSelectionModal = (
   props: WellSelectionModalProps
 ): JSX.Element | null => {
-  const { isOpen, labwareId, onCloseClick, pipetteId } = props
+  const { isOpen, labwareId, onCloseClick, pipetteId, nozzleType } = props
```

Comment:
maybe set the default value?
```suggestion
  const { isOpen, labwareId, onCloseClick, pipetteId, nozzleType=null } = props
```